tests/dbgp/dbgpclient.php

```diff
@@ -60,8 +93,7 @@ function doRead( $conn )
 			$read = preg_replace( '@(2002-20[0-9]{2})@', '2002-2099', $read );
 			echo $read, "\n\n";
 
-			if ( preg_match( '@<stream xmlns="urn.debugger_protocol_v1" xmlns:xdebug@', $read ) )
-			{
+			if ( preg_match( '@<stream xmlns="urn.debugger_protocol_v1" xmlns:xdebug@', $read ) ) {
```

Comment:
Yeah - sorry for asking! In general, when making a PR for a new feature (for any sort of project), it's wise to not change existing CS. It makes it harder to review the actual code changes.